data/keys

```diff
@@ -7,10 +7,10 @@ Global {
 # Adding your own frequently used actions is easy -
 # just copy it over from CHAINS and edit the keypress!
 	# Moving in frames
-	KeyPress = "Mod1 Tab" { Actions = "NextFrame EndRaise" }
-	KeyPress = "Mod1 Shift Tab" { Actions = "PrevFrame EndRaise" }
-	KeyPress = "Mod1 Ctrl Tab" { Actions = "NextFrameMRU EndRaise" }
-	KeyPress = "Mod1 Ctrl Shift Tab" { Actions = "PrevFrameMRU EndRaise" }
+	KeyPress = "Mod4 Tab" { Actions = "NextFrameMRU TempRaise" }
```

Comment:
the mod4 here is a typo right? I agree ..MRU seems to be more common for alt-tab nowadays

Reply:
Here I think we could use a var like MRU which either can be empty or set to MRU so all default bindings can be changed by setting a var. Seems reasonable?

Reply:
@pclouds Yes, it was a typo. I fixed it just now.

@pekdon Yes, that would be a way to make it more easily customizable.

In general, it also very much depends on which keybindings people actually use. I don't use window grouping currently, so I don't use the related bindings. I don't use relative workspace actions, only absolute ones. And I also completely removed the keychains because I don't use them.

Maybe we should group the different kinds of bindings in sections with an descriptive header, so people can easily spot the difference and decide which ones they want.